### PR TITLE
Multiple debts. Multiple suppressions

### PR DESCRIPTION
**Pull request by rogerball**

### CODE REVIEW

File: src/test/resources/features/ifs/Supression.feature [hunk 1]
```diff
@@ -200,4 +233,44 @@ Feature: Suppression
...
+      | ruleId | mainTrans | suppressionIds |
+      | 1      | 1546      | 1              |
+      | 1      | 1535      | 1              |
+       Given suppression data has been created
+      | suppressionId | reason     | description | enabled | fromDate   | toDate     |
+      | 3             | PERIOD-END | desc-3      | true    | 2021-06-01 | 2021-06-20 |
+    And suppression rules have been created
+      | ruleId | periodEnd  | suppressionIds |
+      | 1      | 2019-05-20 | 3              |
+    Given suppression data has been created
```
Comment (aasheishmody) [inline src/test/resources/features/ifs/Supression.feature]: Should this not be 'And', as best practice says you should not be ideally using 'Given' twice in the same scenario.

File: src/test/scala/uk/gov/hmrc/test/api/requests/SuppressionRulesRequests.scala [hunk 2]
```diff
@@ -28,9 +28,9 @@ import uk.gov.hmrc.test.api.utils.{BaseRequests, TestData}
 
 object SuppressionRulesRequests extends ScalaDsl with EN with Eventually with Matchers with BaseRequests {
 
-  def postSuppressionData(json: String): StandaloneWSResponse = {
+  def postSuppressionData(json: String, id: String): StandaloneWSResponse = {
     val bearerToken = createBearerToken(enrolments = Seq("read:suppression-data"))
-    val baseUri     = s"$interestForecostingApiUrl/suppressions/1/suppression"
+    val baseUri     = s"$interestForecostingApiUrl/suppressions/" + id + "/suppression"
```
Comment (thadeusssewanyana) [inline src/test/scala/uk/gov/hmrc/test/api/requests/SuppressionRulesRequests.scala]: maybe we can use string interpolator here like s"$interestForecostingApiUrl/suppressions/$id/suppression"

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/runner/InterestForecastingApiTestRunner.scala [hunk 3]
```diff
@@ -24,6 +24,6 @@ import org.junit.runner.RunWith
   features = Array("src/test/resources/features/ifs"),
   glue = Array("uk.gov.hmrc.test.api.cucumber.stepdefs", "uk.gov.hmrc.test.api.cucumber.hooks"),
   plugin = Array("pretty", "html:target/cucumber", "json:target/cucumber.json"),
-  tags = Array("")
+  tags = Array("@runMe")
```
Comment (thadeusssewanyana) [inline src/test/scala/uk/gov/hmrc/test/api/cucumber/runner/InterestForecastingApiTestRunner.scala]: Do we need this tag ?

File: src/test/resources/features/ifs/Supression.feature (hunk 1 above)
Comment (rogerball) [reply src/test/resources/features/ifs/Supression.feature]: good point. thanks

**Review by ashnewsuk — APPROVED** [review]

**Review by thadeusssewanyana — APPROVED** [review]

**Pull request merged**